src/runtime/runtime.cs

```diff
@@ -147,6 +147,19 @@ public class Runtime
 
         private static PyReferenceCollection _pyRefs = new PyReferenceCollection();
 
+        internal static int pyversionnumber
+        {
+            get
+            {
+                var versionString = PythonEngine.Version;
+                //"3.0a5+ (py3k:63103M, May 12 2008, 00:53:55) \n[GCC 4.2.3]"
+
+                //we only support python 3 so we just need to check the third character
+                return 30 + Convert.ToInt32(versionString[2]);
```

Comment:
This seems very brittle to me, there must be a better solution. Also, we don't support any Python version before 3.5, so checks for that can go.

Reply:
@filmor I could use sys.version_info but I'd rather not run python code.  Let me look for a C API equivalent

Reply:
@filmor see now